tao/BUILD

```diff
@@ -91,30 +102,43 @@ cc_library(
     ],
     hdrs = glob(
         include = [
-            "tao_bridge/**/*.h"
+            "tao_bridge/**/*.h",
         ] + [
             "third_party/json/single_include/nlohmann/json.hpp",
         ],
         exclude = [
             "tao_bridge/gpu/bace_gemm_thunk.h",
         ],
     ),
-    deps = [
-        ":version_header",
-        "@local_config_tf//:tf_header_lib",
-        "@local_config_tf//:libtensorflow_framework",
-        "@org_tao_compiler//tensorflow/compiler/mlir/xla/ral:ral_bridge",
-    ],
-    copts = if_cuda(["-DGOOGLE_CUDA=1"
-        ]) + if_dcu(["-DTENSORFLOW_USE_DCU=1"
-        ]) + if_rocm(["-DTENSORFLOW_USE_ROCM=1",
-            "-x rocm",
-        ]) + tf_copts(),
+    copts = if_cuda([
+        "-DGOOGLE_CUDA=1",
+    ]) + if_dcu([
+        "-DTENSORFLOW_USE_DCU=1",
+    ]) + if_rocm([
+        "-DTENSORFLOW_USE_ROCM=1",
+        "-x rocm",
+    ]) + tf_copts(),
     linkopts = [
         "-lm",
     ],
-    alwayslink = 1,  # targets depending on it should carry all symbols in its children.
     linkstatic = 1,
+    deps = [
+        ":version_header",
+        "@org_tao_compiler//tensorflow/compiler/mlir/xla/ral:ral_bridge",
+    ] + if_tf_serving(
+        [
+            "@org_tensorflow//tensorflow/core:lib",
+            "@org_tensorflow//tensorflow/core:framework",
+            "@org_tensorflow//tensorflow/core/common_runtime:core_cpu",
+            "@org_tensorflow//tensorflow/core/platform:stream_executor_no_cuda",
+            "@org_tensorflow//tensorflow/stream_executor:stream_executor_impl",
```

Comment:
Does this `org_tensorflow` come from `tf_community` ? If so, `tao_bridge` is not built against expected tf_serving version?

Reply:
Tensorflow source code from `tf_community` or release version tgz from `tf_http_archive` can both be `org_tensorflow`, when building against `tf_serving`, the one from `tf_community` will not be used.